File: Java/21. Strong Number.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the Number to check Strong:");
        int enteredNumber = userInput.nextInt();
        int originalNumber = enteredNumber;
        int add = 0;

        while(enteredNumber != 0) {
            int digit = enteredNumber % 10;
            int factorial = 1;
            for(int i = 1; i <= digit; i++) {
                factorial *= i;
            }
            add += factorial;
            enteredNumber /= 10;
        }
        if( add == originalNumber ) {
            System.out.println(originalNumber + " is a Strong number");
        } else {
            System.out.println(originalNumber + " is not a Strong Number");
        }
        userInput.close();
    }
}

File: Python/01. Important Fundamentals/21. Strong Number.py
number = int(input("Enter a number to check Strong:"))
originalValue = number
add = 0
while(number != 0):
    digit = number % 10
    fact = 1

    for i in range(1,digit):
        fact *= i
    add += fact
    number //= 10

if(add == originalValue):
    print(originalValue, "is a Strong Number")
else:
    print(originalValue,"is not a Strong Number")